common/logger.h

```diff
@@ -6,12 +6,13 @@
  * "LICENSE" at the root of this distribution.
  */
 
-#ifndef ACCELERATOR_LOGGER_H_
-#define ACCELERATOR_LOGGER_H_
+#ifndef COMMON_LOGGER_H_
+#define COMMON_LOGGER_H_
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+#include "common/ta_errors.h"
```

Comment:
How about renaming it to `common/errors.h`?

Reply:
I don't think prefix `ta_` shall be applied in header files.

Reply:
The error code of entangled has the same path and it is also named as `errors.h`, so if we remain the error code in `errors.h` conflicts would happen.